core/vm/contracts.go

```diff
@@ -34,6 +34,10 @@ import (
 	"golang.org/x/crypto/ripemd160"
 )
 
+var (
+	errPrecompileDisabled = errors.New("sha256, ripemd160, blake2f precompiles temporarily disabled")
```

Comment:
Caveat: When I `cast call` the precompile directly I see this error. However when I call from another contract I just see `execution reverted`. Not sure why.

Reply:
we can track this on asana?

Reply:
@zimpha found that the revert reason is only propagated if the call returns `err=vm.ErrExecutionReverted` and the revert reason is encoded in `ret.` While that might provide better UX, the current solution is consistent with other precompile errors (like `errBadPairingInput`), and it's problematic to return `ErrExecutionReverted` because of a circular dependency, so let's keep it as it is now.